ui-v2/app/services/services.js

```diff
@@ -21,6 +21,12 @@ export default Service.extend({
       .then(function(item) {
         const nodes = get(item, 'Nodes');
         const service = get(nodes, 'firstObject');
+        const tags = nodes
+          .reduce(function(prev, item) {
+            return prev.concat(get(item, 'Service.Tags') || []);
+          }, [])
+          .uniq();
+        set(service, 'Tags', tags);
```

Comment:
It would be better if this computation could be expressed as a computed property somewhere. I think anyone who went looking for where Tags gets computed would be disappointed to discover it isn't in the model code but is instead in this service.